src/App.tsx

```diff
@@ -55,6 +56,7 @@ export default class App extends React.Component<IAppProps> {
     }
 
     public render() {
+        console.info(`Running VoTT v${appInfo.version}`);
```

Comment:
Added for prod use or just for local debugging?  Since we already have it in the App Settings page do we need it here?

Reply:
it's definitely not necessary; was going to fold in more info here (commit, build date, etc.) but turned out to be challenging w/ Webpack. I'm happy to yank. I do like seeing what version I'm using w/ dev tools, but since we're not bumping patch (or something else...) on each PR, it's less useful.